.github/workflows/.windows-build.yml

```diff
@@ -0,0 +1,100 @@
+# reusable workflow
+name: .windows-build
```

Comment:
Why can't we just keep this in `.windows.yml`? We already have a bunch of workflows, would be nice to avoid unnecessary split.